testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestTablePartitioningWithSpecialChars.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.hive;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.tempto.ProductTest;
+import io.trino.tempto.assertions.QueryAssert;
+import org.testng.annotations.Test;
+
+import static io.trino.tempto.assertions.QueryAssert.Row.row;
+import static io.trino.tempto.assertions.QueryAssert.assertThat;
+import static io.trino.tests.product.utils.QueryExecutors.onHive;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static java.lang.String.format;
+
+public class TestTablePartitioningWithSpecialChars
+        extends ProductTest
+{
+    private static final String INSERTED_PARTITION_VALUES =
+            "(1, 'with-hyphen')," +
+            "(2, 'with.dot')," +
+            "(3, 'with:colon')," +
+            "(4, 'with/slash')," +
+            "(5, 'with\\\\backslash')," +
+            "(6, 'with=equal')," +
+            "(7, 'with?question')," +
+            "(8, 'with!exclamation')," +
+            "(9, 'with%%percent')," +
+            "(10, 'with space')";
+
+    private static final ImmutableList<QueryAssert.Row> EXPECTED_PARTITION_VALUES = ImmutableList.of(
+            row(1, "with-hyphen"),
+            row(2, "with.dot"),
+            row(3, "with:colon"),
+            row(4, "with/slash"),
+            row(5, "with\\\\backslash"),
+            row(6, "with=equal"),
+            row(7, "with?question"),
+            row(8, "with!exclamation"),
+            row(9, "with%percent"),
+            row(10, "with space"));
+
+    @Test
```

Comment:
Since no groups are defined I think these will run in `suite-1` (since suite-1 is the only one that only has excludes, no explicit includes)?

Reply:
Good point - forgot about groups. I will assign @Test(groups = HIVE_PARTITIONING)